setup.py

```diff
@@ -103,6 +109,15 @@
     Executable("start.py", base=None,
                target_name="openpype_console", icon=icon_path.as_posix())
 ]
+if IS_LINUX:
+    executables.append(
+        Executable(
+            "linux_app_launcher.py",
```

Comment:
I would call it just `app_launcher` as it will be generated only on linux boxes so it's little bit overhead